cypress/support/helper.js

```diff
@@ -1,7 +1,4 @@
-import {
-  knobsTab,
-  getKnobsInputWithName,
-} from "../locators";
+import { knobsTab, getKnobsInputWithName } from "../locators";
```

Comment:
Maybe these should be renamed to not reference knobs anymore

Reply:
fixed

Reply:
Looks the same to me 👀